proto/iotics/api/circle.proto

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2019-2024 Iotic Labs Ltd. All rights reserved.
+
+// Iotics Web protocol definitions (circles)
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "iotics/api/common.proto";
+package iotics.api;
+
+option csharp_namespace = "Iotics.Api";
+option go_package = "github.com/Iotic-Labs/iotic-go-proto-qapi/iotics/api;ioticsapi";
+option java_multiple_files = true;
+option java_outer_classname = "CircleProto";
+option java_package = "com.iotics.api";
+option objc_class_prefix = "IAX";
+option php_namespace = "Iotics\\Api";
+
+// ---------------------------------------------------------------------------------------------------------------------
+
+
+// CircleAPI enables creation and management of Iotics circles.
+// Services only affect local resources.
+service CircleAPI {
+
+  // UpsertCircle creates or update a circle with its metadata.
+  // The full state is applied (ie. if the operation succeeds the state of the circle will
+  // be the one described in the payload)
+  rpc UpsertCircle(UpsertCircleRequest) returns (UpsertCircleResponse) {}
+
+  // DeleteCircle deletes a circle.
+  rpc DeleteCircle(DeleteCircleRequest) returns (DeleteCircleResponse) {}
+
+  // Describes a circle.
```

Comment:
(pedantic) Based on how the other ones are written, should this and the below one say e.g.:
"DescribeCircle describes a circle"?

Reply:
Before this was mandatory due to the previous linter we were using. But this information is redundant and annoying to maintain.